README.md

```diff
@@ -105,4 +105,23 @@ Running in separate terminals:
 We are using dxdao-contracts as submodule. Currently it is working under `feature/monorepo-setup-w-create2` branch. 
 After [this pr](https://github.com/DXgovernance/dxdao-contracts/pull/294) is merged we can change .gitmodules config to use dxdao-contracts with main/develop branch
 To update dxdao-contracts submodule to latest commit use `git submodule update --remote --merge`
-To update dxdao-contracts submodule branch use `git submodule set-branch --branch [branch] dxdao-contracts`
\ No newline at end of file
+To update dxdao-contracts submodule branch use `git submodule set-branch --branch [branch] dxdao-contracts`
+
+# DAVI Monorepo QA tests
+
+## How to setup
+
+1. Install dependencies `pnpm i`
+2. Make an `.env` file in the `davi` folder (see `.env.qa.example`)
+
+### How to run test
+
+To run test just type in terminal `yarn test:testName`
```

Comment:
Hm, I think we should stick to using `pnpm` as our package manager. I am not sure if there are any conflicts that could arise with having two separate package managers (`yarn` and `pnpm`). 

So basically test the same way you have but use `pnpm` instead :). https://pnpm.io/cli/test